gpflow/utilities/utilities.py

```diff
@@ -44,12 +44,16 @@ def to_default_float(x):
     return cast(x, dtype=default_float())
 
 
-def set_trainable(model: tf.Module, flag: bool):
+def set_trainable(model: Union[tf.Module, Iterable[tf.Module]], flag: bool) -> None:
```

Comment:
One note while I'm looking at this: `flag` seems like a pretty bad argument name (it doesn't say anything), whereas `model` is too specific (it works on any mod*ule*, e.g. a kernel by itself)...

Reply:
changing names isn't backwards compatible. if you're ok with that I'll change them

Reply:
I'd be happy with changing the names, but you don't have to change them in this PR.

I'm just thinking these are awful names to pass by name.